src/quo2/components/wallet/account_overview/view.cljs

```diff
@@ -0,0 +1,69 @@
+(ns quo2.components.wallet.account-overview.view
+  (:require [quo2.components.icon :as icons]
+            [quo2.components.markdown.text :as quo2]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [utils.i18n :as i18n]
+            [quo2.components.wallet.account-overview.style :as style]))
+
+(defn account-overview
+  [{:keys [account-name current-value bottom-time-text bottom-time-to-text show-details?
+           percentage-change currency-change up?]}]
+  [rn/view
+   {:style style/account-overview-wrapper}
+   [rn/view style/row
+    [quo2/text
+     {:weight :medium
+      :size   :paragraph-1
+      :style  style/account-name}
+     account-name]
+    (when show-details?
+      [icons/icon :i/reveal
+       {:container-style style/reveal-icon
+        :color           (colors/theme-colors colors/neutral-80-opa-40 colors/white-opa-40)}])]
```

Comment:
if you're using `color/theme-colors` you need `theme` as the 3rd param. Eventually we will remove the internal checks in that function to the global ratom for theme.

so then you'll also need to wrap this component in the `with-theme` hoc

Reply:
Done